src/app/page.tsx

```diff
@@ -1,11 +1,32 @@
 'use client'
-import { Heading, Text, VStack } from '@chakra-ui/react'
+import { Heading, Text, Box } from '@chakra-ui/react'
 
 export default function Home() {
+  const outerBoxStyles = {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: '100vw',
+    height: '90vh',
+    p: '25',
+    background:
+      'url(https://images.unsplash.com/photo-1628258334105-2a0b3d6efee1?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=774&q=80) center/cover no-repeat',
```

Comment:
@drewpate this is awesome, great job! 🔥 

Just tiny nit: at certain viewport widths, some of the text can start to blend in with the background image.  Maybe a semi-transparent layer to darken the image would be nice?  Check out the screenshots below and let me know what you think.

<details>
<summary>Screenshots</summary>
  
#### Before
<img width="478" alt="Screenshot 2023-07-01 at 9 33 19 AM" src="https://github.com/mitxpro-dev/learnjs/assets/96446064/c0e53ef9-f184-4e3a-b455-226aec15f061">

#### After
<img width="491" alt="Screenshot 2023-07-01 at 9 33 53 AM" src="https://github.com/mitxpro-dev/learnjs/assets/96446064/bf1a7b93-20e8-4308-b059-72a607c32ead">

</details>

```suggestion
      'linear-gradient(rgba(0, 0, 0, 0.45), rgba(0, 0, 0, 0.45)), url(https://images.unsplash.com/photo-1628258334105-2a0b3d6efee1?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=774&q=80) center/cover no-repeat',
```

Reply:
A potential enhancement to consider:  
- If we could extract the image `url` and declare below the imports as a constant (i.e. `URL_HERO_IMAGE`) but leave the styling in-place.  
- This would enable us to easily change out the hero image without worrying about affecting anything else.

Something like this below the imports:
```
const URL_HERO_IMAGE = 'https://images.unsplash.com/photo-1628258334105-2a0b3d6efee1?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=774&q=80';
```

Then:
```
background: `linear-gradient(rgba(0, 0, 0, 0.45), rgba(0, 0, 0, 0.45)), url(${URL_HERO_IMAGE}) center/cover no-repeat`,
```